materialize-google-pubsub/transactor.go

```diff
@@ -27,9 +34,11 @@ func (t *transactor) Store(it *pm.StoreIterator) error {
 	errGroup, ctx := errgroup.WithContext(it.Context())
 
 	for it.Next() {
-		topic := t.topics[it.Binding]
-		res := topic.Publish(ctx, &pubsub.Message{
-			Data:        it.RawJSON,
+		binding := t.bindings[it.Binding]
+		res := binding.topic.Publish(ctx, &pubsub.Message{
+			Data: it.RawJSON,
+			Attributes: map[string]string{
+				IDENTIFIER_ATTRIBUTE_KEY: fmt.Sprintf("%s/%s", binding.topicName, binding.identifier)},
```

Comment:
Would it be possible to have the default identifier value be `""` and then only include the identifier attribute if it's non-empty? That seems like it might be preferable for the simple use-cases.

Reply:
Great idea, I updated the code to not include the attribute if the identifier is empty.